search-service/src/main/java/de/cxp/ocs/SearchController.java

```diff
@@ -51,7 +51,6 @@
 @CrossOrigin("*")
 @RestController
 @RequestMapping(path = "/search-api/v1")
-@EnableScheduling
```

Comment:
@HJK181 @gabauer this made the scheduler be enabled all the time - actually it should be configurable